pkgs/applications/science/misc/simgrid/default.nix

```diff
@@ -78,12 +109,9 @@ stdenv.mkDerivation rec {
          scheduling on distributed computing platforms ranging from simple
          network of workstations to Computational Grids.
       '';
-
     homepage = http://simgrid.gforge.inria.fr/;
-
-    license = stdenv.lib.licenses.lgpl2Plus;
-
-    maintainers = [ ];
-    platforms = stdenv.lib.platforms.gnu;  # arbitrary choice
+    maintainers = with maitainers; [ mickours ];
```

Comment:
Typo. Probably whats causing the evaluation to fail.

Reply:
sorry about that, fixed by 42ec455